client/src/main/java/io/spine/client/TargetBuilder.java

```diff
@@ -42,32 +42,28 @@
  * <p>The {@link Target} matching the builder configuration is created with {@link #buildTarget()},
  * while the {@link FieldMask} is composed with {@link #composeMask()}.
  *
- * <p>The public API of this class is inspired by SQL syntax.
- * <pre>
- *     {@code
+ * <p>The public API of this class is inspired by SQL syntax:
+ * <pre>{@code
```

Comment:
Do we still use `{@code }` for multiline snippets? According to @armiol, it might not be necessary. Let's discuss.

Reply:
It's not just pre-formatted something (like console example, for example). It is code.